snap/snapcraft.yaml

```diff
@@ -147,12 +149,12 @@ parts:
       initrd_distro=$(${yq} r -X ${kata_dir}/versions.yaml assets.initrd.architecture.${arch}.name)
       image_distro=$(${yq} r -X ${kata_dir}/versions.yaml assets.image.architecture.${arch}.name)
       case "$arch" in
-        x86_64)
+        x86_64|aarch64)
```

Comment:
Nit: Might be good to make the order alphabetical.

Reply:
Okay